app/src/app/players/[username]/competitions/page.tsx

```diff
@@ -39,15 +46,27 @@ export default async function PlayerCompetitionsPage(props: PageProps) {
 
   const mappedCompetitions = competitions.map((c) => c.competition);
 
-  const ongoing = mappedCompetitions.filter(
-    (c) => getCompetitionStatus(c) === CompetitionStatus.ONGOING
-  );
+  const ongoing = mappedCompetitions
```

Comment:
why not just use `ongoingCompStandings`? they include the competition's data, so you wouldn't need `mappedCompetitions` for ongoing/finished competitions data

Reply:
done